src/react-components/popover/ImageGridPopover.js

```diff
@@ -19,6 +19,7 @@ export function ImageGridPopover({ fullscreen, items, closePopover }) {
 
               closePopover();
             }}
+            {...item.crossOrigin && { crossOrigin: item.crossOrigin }}
```

Comment:
React doesn't pass in attributes that are not truthy, so I'd write this line as: 
```
crossOrigin={item.crossOrigin ? item.crossOrigin : false}
```

Reply:
Though if this ImageGridPopover is used _only_ for the emojis, you can just use

`crossOrigin="anonymous"` for this line, and remove using props altogether.

Reply:
Good question. I just wanted to add flexibility to `ImageGridPopover` because there might be a chance that `ImageGridPopover` might be reused somewhere else and we might want not to send crossOrigin from it (for the security reason?) in the future.

But it's no my big preference. If we don't need to take care of currently unnecessary flexibility, hard-coded `crossOrigin="anonymous"` would be good.

Reply:
Any thoughts on this? @robinkwilson @brianpeiris @netpro2k

Reply:
@takahirox I forgot to submit the above comment...